crates/astria-sequencer/src/mempool.rs

```diff
@@ -126,68 +208,107 @@ impl std::hash::Hash for EnqueuedTransaction {
 /// - transaction expiration
 #[derive(Clone)]
 pub(crate) struct Mempool {
-    inner: Arc<RwLock<MempoolQueue>>,
+    queue: Arc<RwLock<MempoolQueue>>,
+    set: Arc<RwLock<HashSet<[u8; 32]>>>,
+    invalid_cache: Arc<RwLock<InvalidCache>>,
+    tx_ttl: i64,
```

Comment:
do we need to store this if it's already a constant? it appears to always be set to the same value